assets/scss/components/_toggle-button.scss

```diff
@@ -2,28 +2,59 @@
 Toggle Button
 
 Markup:
-<span class="toggle-btn {{modifier_class}}">Toggle Button</span>
```

Comment:
Is your service the only place this is being used? Can we be sure it's not going to break other frontends?